src/Resources/config/services.yml

```diff
@@ -0,0 +1,8 @@
+services:
+    ContaoCommunityAlliance\Translator\Contao\ContaoTranslatorService:
+        arguments:
+            - '@event_dispatcher'
+
+    contao.translator:
```

Comment:
Definitely our vendor prefix.

I agree that we must discuss the naming conventions to use in our repositories.